{"name": "测试用例描述：错误的密码登录失败", "status": "passed", "description": "测试描述：错误的密码登录失败", "steps": [{"name": "输入正确账号", "status": "passed", "attachments": [{"name": "正确账号", "source": "89d2200a-c828-4afc-87cd-a9ce719ec149-attachment.attach"}], "start": 1651818001581, "stop": 1651818001582}, {"name": "输入错误密码", "status": "passed", "attachments": [{"name": "错误密码", "source": "c22c829a-3f82-47bb-b75f-911b2aaebfde-attachment.attach"}], "start": 1651818001657, "stop": 1651818001657}, {"name": "输入正确验证码", "status": "passed", "attachments": [{"name": "正确验证码", "source": "ac511d26-61a8-40f4-8bd1-a012707c6c6a-attachment.attach"}], "start": 1651818001703, "stop": 1651818001703}], "attachments": [{"name": "截图", "source": "8aeb0bca-52ff-4375-b9bb-ca997d1f9ee2-attachment.png", "type": "image/png"}], "start": 1651818001581, "stop": 1651818004034, "uuid": "fe604734-aef5-4f9d-8503-a16347c633ee", "historyId": "8ec91ebb07cb2b45a38478eca1be69f5", "testCaseId": "a86650c2206e92995fa9f8bad029494f", "fullName": "test_two.Test#test_06", "labels": [{"name": "epic", "value": "xxx测试"}, {"name": "story", "value": "登录"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "登录模块"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "2072-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}